### ISSUE TITLE
GitHub access rights

### ISSUE THREAD

**Issue by larsrh** [typelevel/general#65]
Our current access rights are what one would usually call "historically grown", or, in simpler terms, "all over the place". There are teams, owners, and outside collaborators, with no clear structure. I'd like to fix this and also set precedent for further decisions in that vein.

_Note that this only applies to projects hosted below the @typelevel GitHub organisation._ All other projects are free to legislate as they wish.

## General ideas

- I'd like to suggest everyone to set their membership status to "public" to increase transparency. This is an individual decision though, and I don't suggest making this a policy. In case you'd like to do that, go to [the members page](https://github.com/orgs/typelevel/people) and flip the switch.
- Also, everyone should consider enabling 2FA.

## Owners

The following people are currently owners:

- @folone 
- @larsrh
- @milessabin 
- @non 
- @stew 
- @tixxit

Not all of them exercise administrative rights on a regular basis (e.g. moving repositories). I have no concrete suggestion to change the set of people here; I just wanted to document the status quo.

## Legacy teams

There are a couple of teams left.

- The `Core` team is equivalent to the set of owners and didn't grant any additional permissions, so I've deleted it.
- Two teams had only one member: `scala` (@propensive) and `shapeless-contrib` (@StacyCurl). I've converted those to "outside collaborators", i.e. granting them write access to the respective repositories and deleted the then-empty teams.
- Left are the two teams `typelevel.org` (5 members) and `Cats` (12 members). I suggest proceeding in the same manner as the above (converting to outside collaborators, then deleting).

The reason why I suggest converting to teams to outside collaborators is because the latter are much easier to administrate and also allow for some form of self-governance within a project (i.e. there can be project admins which can grant access to more people without themselves being owners of @typelevel).

**Comment by larsrh** [comment]
I just realized that the `Cats` team is also superfluous because it is a subset of the people who are also outside collaborators with write access to the repository. Deleting that one too, then.

**Comment by propensive** [comment]
Just commenting to check what word now appears up here                             ^^^^^^^^^^^^^^^^^^^

**Comment by larsrh** [comment]
@propensive You're apparently not a member anymore now; GitHub offers an "invite to organization" button but I'm not sure what that does actually.

**Comment by propensive** [comment]
Hmmm. I'm also bit confused by the terminology here. You're also being displayed as "Member", though shouldn't you be "Owner"?

**Comment by larsrh** [comment]
Yeah, I have no idea ¯\_(ツ)_/¯

**Comment by larsrh** [comment]
I've now also deleted the last remaining team, with hopefully less noise all around.

**Comment by rossabaker** [comment]
Well, I went back to teams.  See also https://github.com/typelevel/governance/issues/14
